media/js/invokeRemoteRestApiVue.js

```diff
@@ -0,0 +1,68 @@
+console.log('Loaded!')
```

Comment:
I know this is in the other one, but where does this actually get logged to?

Reply:
I think the "browser" (or, equivalent developer console)

Reply:
removed